jekyll/assets/css/footer.css

```diff
@@ -2,6 +2,8 @@
  * Footer
  * */
 .footer {
+  position: relative;
```

Comment:
Are there other styles that are making us require this? I tried removing it and didn't see many changes 😄 From what I read on [css tricks](https://css-tricks.com/almanac/properties/p/position/), I _think_ `relative` is the same as the default `static`, except it lets you do positioning with `top`/`left` etc.

In my experience, I've only used `position: relative` when I'm also messing around with `position: absolute`